content/intro/pricing/rates.md

```diff
@@ -18,44 +18,22 @@ Find out [more about this pricing model]({{< relref "intro/pricing/pricing-model
 
 Packages are selected based on the kinds of applications to be hosted.
 
-<table class="responsive_table">
-  <tr>
-    <th width="25%"><span>Sandbox</span></th>
-    <th width="25%"><span>Prototyping</span></th>
-    <th width="25%"><span>FISMA Low</span></th>
-    <th width="25%"><span>FISMA Moderate</span></th>
-  </tr>
-  <tr>
-    <td valign="top">
-      For any .gov or .mil account to try out cloud.gov, no paperwork required. All managed services are available.
-    </td>
-    <td valign="top">
-      Sandbox usage caps can be raised. Suitable for many teams to deploy apps, though limited to the apps.cloud.gov domain. Low ability to delegate access control to teams. No production data allowed. Usually purchased per agency/department. 
-    </td>
-    <td valign="top">
-      One public-facing FISMA Low system, including all the spaces needed and DNS support.
-    </td>
-    <td valign="top">
-      One public-facing FISMA Moderate system, including all the spaces needed and DNS support. Additional support for FISMA Moderate data requirements.
-    </td>
-  </tr>
-  <tr>
-    <td>Free</th>
-    <td>$15K / year plus usage charges</th>
-    <td>$20K / year plus usage charges</th>
-    <td>$90K / year plus usage charges</th>
-  </tr>
-</table>
-
-Access package fees are charged yearly and are non-severable. 
+| Package | What's included? | Base annual fee | Usage charges? |
+| --- | --- | --- | --- |
+| **Sandbox** | Any user with a `.gov` or `.mil` email address can try out cloud.gov, no paperwork required. All managed services are available. Resource usage is capped, though it can be raised. | Free | No |
```

Comment:
"Sandbox usage caps can be raised" used to be listed under the `Prototyping` package...not sure if that was intentional?